docs/watch-examples.md

```diff
@@ -215,7 +215,7 @@ Likely because of an enthusiastic pattern trying to watch thousands of files. Su
 Another reason if you're watching a large number of files could be the low default `interval`. Try increasing with `options: { interval: 5007 }`. Please see issues [#35](https://github.com/gruntjs/grunt-contrib-watch/issues/35) and [#145](https://github.com/gruntjs/grunt-contrib-watch/issues/145) for more information.
 
 ## Why spawn as child processes as a default?
-The goal of this watch task is as files are changed, run tasks as if they were triggered by the user themself. Each time a user runs `grunt` a process is spawned and tasks are ran in succession. In an effort to keep the experience consistent and continually produce expected results, this watch task spawns tasks as child processes by default.
```

Comment:
This seems correct. The grammar before this change could use some work though. I'd get rid of `The goal of this watch task is`.